File: ITManagementClient/Models/ResponseModels/Workers/UpdateWorkerResponseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ITManagementClient.Models.ResponseModels.Workers
{
    public class UpdateWorkerResponseModel
    {
        public int WorkerId { get; set; }

        public int Position { get; set; }

        public string Name { get; set; }

        public decimal Salary { get; set; }

        public DateTime BirthDate { get; set; }

        public DateTime HireDate { get; set; }

        public bool Active { get; set; }

        public string EnglishLevel { get; set; }

        public string Login { get; set; }
    }
}

File: ITManagementServer/src/main/java/models/responseModels/workers/UpdateWorkerResponseModel.java
package models.responseModels.workers;

import java.sql.Date;

public class UpdateWorkerResponseModel {
    private int WorkerId;

    private int Position;

    private String Name;

    private double Salary;

    private Date BirthDate;

    private Date HireDate;

    private boolean Active;

    private String EnglishLevel;

    private String Login;

    public int getWorkerId() {
        return WorkerId;
    }

    public void setWorkerId(int workerId) {
        WorkerId = workerId;
    }

    public int getPosition() {
        return Position;
    }

    public void setPosition(int position) {
        Position = position;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public double getSalary() {
        return Salary;
    }

    public void setSalary(double salary) {
        Salary = salary;
    }

    public Date getBirthDate() {
        return BirthDate;
    }

    public void setBirthDate(Date birthDate) {
        BirthDate = birthDate;
    }

    public Date getHireDate() {
        return HireDate;
    }

    public void setHireDate(Date hireDate) {
        HireDate = hireDate;
    }

    public boolean isActive() {
        return Active;
    }

    public void setActive(boolean active) {
        Active = active;
    }

    public String getEnglishLevel() {
        return EnglishLevel;
    }

    public void setEnglishLevel(String englishLevel) {
        EnglishLevel = englishLevel;
    }

    public String getLogin() {
        return Login;
    }

    public void setLogin(String login) {
        Login = login;
    }
}
